### PR TITLE
docs: add prerelease doc file

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+> Note: This changelog only includes the changes for the stable versions of VSCode API Center. For the changelog of pre-released versions, please refer to the [VSCode APICenter Changelog](https://github.com/microsoft/vscode-azureapicenter/blob/main/PRERELEASE.md).
```
Comment (formulahendry) [inline CHANGELOG.md:3]: 'VSCode API Center' -> 'Azure API Center for Visual Studio Code'
[VSCode APICenter Changelog] -> [Pre-release Changelog of Azure API Center for Visual Studio Code]

File: PRERELEASE.md
Comment (formulahendry) [inline PRERELEASE.md:1]: Please also help add header section, refer to https://github.com/OfficeDev/teams-toolkit/blob/dev/packages/vscode-extension/PRERELEASE.md

File: CHANGELOG.md (hunk 1 above)
Comment (wenytang-ms) [reply CHANGELOG.md]: updated.

File: PRERELEASE.md
Comment (wenytang-ms) [reply PRERELEASE.md:1]: updated

**Review by formulahendry — APPROVED** [review]

**Pull request merged**